### PR TITLE
Add my image

### PR DESCRIPTION
**Pull request by ppabcd**
Add my image for Google Code-in task.

### CODE REVIEW

**Comment by abishekvashok** [comment]
Please squash the commits :)

File: index.html [hunk 1]
```diff
@@ -294,9 +294,10 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		                    <img class="col-sm-2" alt="Aleksandr Lymarenko" src="img/a-v-lymarenko.png">
 		                    <img class="col-sm-2" alt="Naveen Rajan" src="img/naveen.jpg">
 		                    <img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
+                            <img class="col-sm-2" alt="Reza Juliandri" src="img/reza_juliandri.jpg">
```
Comment (abishekvashok) [inline index.html]: Intending these lines with 7 more lines (to be exact) would be great I guess. Can you do that pls?

**Review by abishekvashok — COMMENTED** [review]
Also squash the commits.

File: index.html (hunk 1 above)
Comment (ppabcd) [reply index.html]: please wait my git result error xd

Comment (abishekvashok) [reply index.html]: @ppabcd don't rush take your own time :)

Comment (ppabcd) [reply index.html]: done :)

Comment (abishekvashok) [reply index.html]: I was referring to the above particular line. @ppabcd

**Review by tstreamDOTh — CHANGES_REQUESTED** [review]
Remove the unnecessary new lines.

**Review by ParthS007 — APPROVED** [review]
Good Work! Please resolve the conflicts and else looks fine to me.

**Comment by ppabcd** [comment]
done

**Comment by NaviRocker** [comment]
Please Squash all the Commits

**Review by isuruAb — APPROVED** [review]
👍

**Comment by isuruAb** [comment]
@ppabcd  please add a descriptive Pull request title and description next time.
cheers 👍

**Comment by ppabcd** [comment]
Thank you very much

**Pull request merged**